src/quickstart/cpp/PdxSerializer.cpp

```diff
@@ -165,15 +165,15 @@ int main(int argc, char** argv) {
     LOGINFO("Registered Person Query Objects");
 
     // Populate the Region with some Person objects.
-    Person* p1 = new Person("John", 1 /*ID*/, 23 /*age*/);
+    Person* p1 = new Person((char *)"John", 1 /*ID*/, 23 /*age*/);
```

Comment:
Those methods pass a string literal to a method whose first parameter is char *. An alternative would be to modify the type of the parameter but that was a more invasive change when the goal was to reduce the clutter in the log that got in the way of determining the root cause.